varcode/vcf.py

```diff
@@ -65,19 +65,21 @@ def load_vcf(
 
     ensembl = EnsemblRelease(release=ensembl_release)
 
-    # We ignore "no-call" variants, i.e. those where X.ALT = [None].
-    variants = [
-        Variant(
-            contig=x.CHROM,
-            pos=x.POS,
-            ref=x.REF,
-            alt=x.ALT[0].sequence,
-            info=x.INFO,
-            ensembl=ensembl)
-        for x in vcf_reader
-        if x.ALT[0] and (not only_passing or not x.FILTER or x.FILTER == "PASS")
-    ]
-
+    variants = []
+    for record in vcf_reader:
```

Comment:
Yep, currently we're only attaching the INFO dict to variants and not the sample info. What do you think is the right way to handle e.g. genotypes? Should we make a nested sample_name -> key -> value dictionary of dictionaries? That seems messy but I'm not sure what else to do. In any case, that should definitely be a separate PR.